src/unk_02074910.c

```diff
@@ -0,0 +1,17 @@
+#include "unk_02074910.h"
+
+BOOL SavArray_IsNatDexEnabled(SAVEDATA* savedata) {
+    return Pokedex_IsNatDexEnabled(Sav2_Pokedex_get(savedata));
+}
+
+BOOL Pokedex_IsNatDexEnabled(const POKEDEX* pokedex) {
+    return (Pokedex_GetNatDexFlag(pokedex) == 1);
+}
+
+u32 sub_02074930(BOOL natDexFlag, u32 species) {
+    if(natDexFlag == 0) {
```

Comment:
Style guide: Insert space between conditional keywords (`if`, `for`, `while`) and opening parenthesis.
Style guide: BOOL params use the TRUE and FALSE macros for their values

```suggestion
    if (natDexFlag == FALSE) {
```